### ISSUE TITLE
Binary classification vs multi-class classification vs Multi-label Classification 

### ISSUE THREAD

**Issue by YunchaoYang** [YunchaoYang/Blogs#38]
--------------------------------------------------------------------------------------
| Binary classification | multi-class classification | Multi-label Classification | Multi-output Classification |
|-----------------------|----------------------------|----------------------------|----------------------------|
|    Sigmoid classifier and BinaryCrossEntropy  Loss  | Softmax classifier and CrossEntropy Loss                           | Sigmoid classifier and BinaryCrossEntropy Loss | Sigmoid classifier and BinaryCrossEntropy Loss 
|-----------------------|----------------------------|----------------------------|

Note the difference between the Multi-label and Multi-output classification:
For Multi-output, each entry of the output has a fixed meaning of 0/1. Each class must choose from 0 and 1. And the number of output is fixed.

For multi-label, you have a pool of output labels, you are asked to choose the proper labels from the label pool that described in the image. The label pool is usually large and abundant.

# reference:
https://learnopencv.com/multi-label-image-classification-with-pytorch-image-tagging/
